README.md

```diff
@@ -0,0 +1,33 @@
+<p align="center">
+<a href="https://http://slackthemes.net" target="_blank"><img src="https://raw.githubusercontent.com/paracycle/slackthemes/master/source/favicon-32.png"></a>
+<br /><strong>Slack Themes</strong>
+</p>
+<p align="center">
+<a href="https://travis-ci.org/paracycle/slackthemes" target="_blank"><img src="https://travis-ci.org/paracycle/slackthemes.svg?branch=master"></a>
+</p>
+
+
+### About the project
+
+slackthemes.net is a platform for browsing and sharing themes for [Slack](https://slack.com/).
+It is not affiliated with [Slack](https://slack.com/) in any way or form; but it is a great tool, use it.
+
+
+### How to contribute (themes)
+
+1. Fork this project.
+2. Add your theme to the `data/themes.yml` file.
+3. Create a pull request.
+
+### How to contribute (core)
+
+1. Fork this project.
+2. Install dependencies using `bundle install`
+3. Make changes.
+4. Build the project using `bundle exec middleman build`
+5. You can run a local webserver to test the project using `php -t ./build -S 0.0.0.0:8080` and access it on `localhost:8080`
```

Comment:
Can we add `python3 -m http.server 8080` and `python -m SimpleHTTPServer 8080` options as well? Most distros come with `python` OOB, and not `php`, so would be great to include.

Failing that, we could redirect them to a better resource like: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server by turning `You can run a local webserver` part into a link.

Take your pick!

Reply:
I changed the webserver text to:

> 5. You can [run a local webserver](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server) to test the project. (make sure to use the `./build` folder as your DocumentRoot).

I tried fitting in all information including the PHP and Python options, but this became way to messy. Since that information is on the Mozilla page anyway, I opted for this solution. 😄